<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1343</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1321</width>
      <height>651</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Error model</string>
     </attribute>
     <widget class="QTableWidget" name="tableInfo">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>1050</x>
        <y>430</y>
        <width>251</width>
        <height>121</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>421</width>
        <height>16777215</height>
       </size>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>120</number>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>29</number>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>15</number>
      </attribute>
     </widget>
     <widget class="PlotWidget" name="graphWidget" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>541</width>
        <height>501</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxArchitecture">
      <property name="geometry">
       <rect>
        <x>1180</x>
        <y>170</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Chip</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Macro</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBoxVNoise">
      <property name="geometry">
       <rect>
        <x>1180</x>
        <y>200</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Correlated</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Uncorrelated</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>1060</x>
        <y>170</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Architecture :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>1060</x>
        <y>200</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Voltage noise :</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxVibMode">
      <property name="geometry">
       <rect>
        <x>1180</x>
        <y>230</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Axial STR</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Axial COM</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>1060</x>
        <y>230</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Vibrational mode :</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>1030</x>
        <y>20</y>
        <width>21</width>
        <height>581</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>1050</x>
        <y>140</y>
        <width>241</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>30</y>
        <width>21</width>
        <height>571</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>20</y>
        <width>57</width>
        <height>271</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>dzB :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Om :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>nuSE :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>SBa :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>SV :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>nuXY :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>nbar :</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>20</y>
        <width>251</width>
        <height>281</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QSlider" name="sliderGradient">
         <property name="minimum">
          <number>24</number>
         </property>
         <property name="maximum">
          <number>200</number>
         </property>
         <property name="singleStep">
          <number>2</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::NoTicks</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="sliderPower">
         <property name="minimum">
          <number>10</number>
         </property>
         <property name="maximum">
          <number>150</number>
         </property>
         <property name="value">
          <number>50</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="sliderENoise">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="sliderBAmbient">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="sliderVNoise">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="sliderNuXY">
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="sliderNbar">
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>20</y>
        <width>81</width>
        <height>271</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="labelGradient">
         <property name="text">
          <string>100 T/m</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelPower">
         <property name="text">
          <string>100 T/m</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelENoise">
         <property name="text">
          <string>100 T/m</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelBAmbient">
         <property name="text">
          <string>100 T/m</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelVNoise">
         <property name="text">
          <string>100 T/m</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelNuXY">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelNbar">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>1100</x>
        <y>60</y>
        <width>151</width>
        <height>76</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QRadioButton" name="radioBtnPulseShaping">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Pulse Shaping</string>
         </property>
         <property name="autoExclusive">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioBtnShowOffRes">
         <property name="text">
          <string>Show</string>
         </property>
         <property name="autoExclusive">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioBtnIncludeOffErr">
         <property name="text">
          <string>Include Error</string>
         </property>
         <property name="autoExclusive">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>1060</x>
        <y>30</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Off-resonant coupling :</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioBtnOptFid">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>1060</x>
        <y>400</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>Optimize fidelity : </string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QSlider" name="sliderFixNu">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>1250</x>
        <y>400</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>1200</x>
        <y>400</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Fix nu :</string>
      </property>
     </widget>
     <widget class="Line" name="line_4">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>300</y>
        <width>431</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxTraceNum">
      <property name="geometry">
       <rect>
        <x>1140</x>
        <y>290</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>One</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Two</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Three</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Four</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>1060</x>
        <y>290</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Trace :</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioBtnTraceUpdate">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>1050</x>
        <y>320</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>Update :       </string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioBtnTraceHide">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>1050</x>
        <y>350</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>Hide :           </string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="Line" name="line_5">
      <property name="geometry">
       <rect>
        <x>1050</x>
        <y>260</y>
        <width>241</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_7">
      <property name="geometry">
       <rect>
        <x>1050</x>
        <y>380</y>
        <width>241</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioBtnAmpNoise">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>320</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>Amplitude noise :</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QSlider" name="sliderChi">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>350</y>
        <width>249</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>24</number>
      </property>
      <property name="maximum">
       <number>200</number>
      </property>
      <property name="singleStep">
       <number>2</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::NoTicks</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>340</y>
        <width>55</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Chi :</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelChi">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>340</y>
        <width>79</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>100 T/m</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioBtnCCWNoise">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>380</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>CCW Current Noise :</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="labelSA">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>400</y>
        <width>79</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>100 T/m</string>
      </property>
     </widget>
     <widget class="QSlider" name="sliderSA">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>410</y>
        <width>249</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>24</number>
      </property>
      <property name="maximum">
       <number>200</number>
      </property>
      <property name="singleStep">
       <number>2</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::NoTicks</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>400</y>
        <width>55</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>SA :</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioBtnSymFluc">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>440</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>Trap frequency fluctuations : </string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="labelSymFluc">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>460</y>
        <width>79</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QSlider" name="sliderSymFluc">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>470</y>
        <width>249</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>24</number>
      </property>
      <property name="maximum">
       <number>200</number>
      </property>
      <property name="singleStep">
       <number>2</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::NoTicks</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>460</y>
        <width>55</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>DeltaS</string>
      </property>
     </widget>
     <widget class="PlotWidget" name="legendWidget" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>540</y>
        <width>541</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Help</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>1261</width>
        <height>521</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Heating</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Decoherence</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Kerr Coupling</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Off-resonant coupling</string>
       </attribute>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1343</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuPresets">
    <property name="title">
     <string>Presets</string>
    </property>
    <addaction name="actionLoadMacro"/>
    <addaction name="actionLoadChip"/>
    <addaction name="separator"/>
    <addaction name="actionLoadPresetFile"/>
    <addaction name="actionSavePresetFile"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPresets"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoadMacro">
   <property name="text">
    <string>Load Macro</string>
   </property>
  </action>
  <action name="actionLoadChip">
   <property name="text">
    <string>Load Chip</string>
   </property>
  </action>
  <action name="actionLoadPresetFile">
   <property name="text">
    <string>Load from File</string>
   </property>
  </action>
  <action name="actionSavePresetFile">
   <property name="text">
    <string>Save to File</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
